superset/assets/src/SqlLab/getInitialState.js

```diff
@@ -12,6 +12,7 @@ export default function getInitialState({ defaultDbId, ...restBootstrapData }) {
     autorun: false,
     dbId: defaultDbId,
   };
+  const { common } = restBootstrapData;
 
   return {
     sqlLab: {
```

Comment:
What if we ever want to add a field here? We'd have to add it outside of this object (like in `common`), and then store it back in `localStorage` by adding it somewhere else right (like in `TabbedSqlEditors.jsx`)?

Reply:
as long as new field is part of `sqlLab`, it will be persisted into localStorage.
After this PR, for all other root level keys in redux state, will not be persisted.